server/py/services/api/tests/unit/crud/test_artifacts.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import time
+import uuid
+
+import sqlalchemy.orm
+
+import mlrun.common.schemas.artifact
+
+import services.api.crud
+
+
+class TestArtifacts:
+    def test_list_artifacts(
```

Comment:
why did you test it in the crud, and not only in the DB?

Reply:
I actually started from here then realized I need to work against mysql (thus integration test fix as well)
test itself has no harm, does it?

Reply:
no harm, just wandered, because you are actually unit-testing the db functionality.